src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -71,18 +70,12 @@ public partial class TextBox : Control, IFrameworkTemplatePoolAware
 
 		public TextBox()
 		{
-			_isPassword = false;
 			InitializeVisualStates();
 			this.RegisterParentChangedCallback(this, OnParentChanged);
 		}
 
 		private void OnParentChanged(object instance, object key, DependencyObjectParentChangedEventArgs args) => UpdateFontPartial();
 
-		protected TextBox(bool isPassword)
```

Comment:
Since this one is not used, you'll need to add `System.Void Windows.UI.Xaml.Controls.TextBox..ctor(System.Boolean isPassword)` to the [DiffIgnore file](https://github.com/unoplatform/uno/blob/master/build/PackageDiffIgnore.xml). There's a block missing since the update to 2.2.0, carl's PR will add it.